terraform/gcp/var.tfvars

```diff
@@ -0,0 +1,30 @@
+# project            = "ia-content"
```

Comment:
This also seems to be unwanted file?

Reply:
Thought this would be a reference file for the user to get to know about the variables usage and values. Tried to keep it inline with AWS var.tfvars file.
Will surely remove it.